### DISCUSSION TITLE
How do you get used to switching back and forth when writing?

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#404] [Q&A]
I am trying to take notes and for the most part I forget I don't use Meow to navigate unless I just opened the file. When I am taking notes and need to move I default back to the arrow keys. Is this something that just takes a bit of time or do others also do the same for something like writing?

**Comment by DogLooksGood** [comment]
Well, it's totally up to you. You can use C-p/n/f/b and other builtin movement commands as well.

**Reply by chookity-pokk** [reply]
Right, I guess I'm moreso asking if people typically hop into Meow movements any time they need to move or do others also default to standard movements? If the former, then how do you get into that mindset?

**Reply by DogLooksGood** [reply]
Well, I don't know others, but by design, working well with builtin commands is one original goal.
